addons-l10n/en/animated-thumb.json

```diff
@@ -1,4 +1,16 @@
 {
   "animated-thumb/set-thumbnail": "Set Thumbnail",
-  "animated-thumb/added-by": "Button added by Scratch Addons browser extension"
+  "animated-thumb/added-by": "Button added by Scratch Addons browser extension",
+  "animated-thumb/description": "Upload a thumbnail from a file, or click \"Use Stage\" to use the current stage as the thumbnail.",
+  "animated-thumb/keep-thumb": "Stop Thumbnail Auto-saving (Recommended)",
+  "animated-thumb/keep-thumb-desc": "This stops the editor from overwriting the uploaded thumbnail with the stage's contents when saving the project.",
+  "animated-thumb/use-stage": "Use Stage",
+  "animated-thumb/select-file": "Select File",
+  "animated-thumb/close": "Close",
+  "animated-thumb/successful": "Thumbnail was uploaded. Note that it takes up to one day before users can see the new thumbnail.",
+  "animated-thumb/server-error": "Thumbnail could not be uploaded due to server error. Please try again later.",
+  "animated-thumb/error": "Thumbnail could not be uploaded.",
+  "animated-thumb/too-big": "The file is too big.",
+  "animated-thumb/gif": "Projects with animated thumbnails will not appear on the front page. Upload anyway?",
```

Comment:
or explore, or search

Reply:
How about "Projects with animated thumbnails may not appear on the front page, the explore page, or in search results."?